internal/common/test/params_file.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2017 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const runfiles = require(process.env['BAZEL_NODE_RUNFILES_HELPER']);
+
+// The arguments are passed via a params file
+const params =
+    require('fs').readFileSync(runfiles.resolve(process.argv[2]), 'utf-8').split(/\r?\n/);
+
+const expected = [
+  'some_value',
+  // $location (expands to runfiles manifest path of format repo/path/to/file)
+  'build_bazel_rules_nodejs/package.json',
+  // $locations (expands to runfiles manifest path of format repo/path/to/file)
+  'build_bazel_rules_nodejs/package.json',
+  'build_bazel_rules_nodejs/internal/common/test/foo/bar/a.txt',
+  'build_bazel_rules_nodejs/internal/common/test/params_file.spec.js',
+];
+
+if (params.length !== expected.length) {
```

Comment:
nit: don't we have some assertion library in the node_modules easily available that does assertEquals on arrays?

Reply:
thanks. switched to jasmine. good call.